src/lib/Storage.svelte

```diff
@@ -13,7 +13,8 @@
   export const chatsStorage = persisted('chats', [] as Chat[])
   export const latestModelMap = persisted('latestModelMap', {} as Record<Model, Model>) // What was returned when a model was requested
   export const globalStorage = persisted('global', {} as GlobalSettings)
-  export const apiKeyStorage = persisted('apiKey', '' as string)
+  const apiKeyFromEnv = import.meta.env.OPENAI_API_KEY || 'some default token'
```

Comment:
Almost good - I would make the default empty string so the current behaviour is not changed.